clients/python/client/osparc/_http_client.py

```diff
@@ -41,3 +44,44 @@ async def __aexit__(self, exc_type, exc_value, traceback) -> None:
                     raise err from exc_value
             await self._client.aclose()
             raise exc_value
+
+    async def _request(
+        self, method: Callable[[Any], Awaitable[httpx.Response]], *args, **kwargs
+    ) -> httpx.Response:
+        @tenacity.retry(
+            reraise=True,
+            wait=tenacity.wait_exponential(
+                multiplier=self.configuration.retries.backoff_factor
+            ),
+            stop=tenacity.stop_after_attempt(
+                self.configuration.retries.total
+                if isinstance(self.configuration.retries.total, int)
+                else 4
```

Comment:
No magic numbers please. Create a constant for this. :-)

Reply:
Right, the total should be an `int` so I think I will just add an assert and avoid adding an extra constant here all together.